modules/salt/pages/yomi.adoc

```diff
@@ -0,0 +1,174 @@
+[[yomi.installer]]
+= Install with Yomi
+
+Yomi (yet one more installer) is an installer for openSUSE operating systems.
+It is a SaltStack state, and can be used for installing Salt clients.
+
+Yomi consists of two components:
+
+* The Yomi formula, which contains the Salt states and modules required to drive the installation.
+For the formula, see https://build.opensuse.org/package/show/systemsmanagement:yomi/yomi-formula.
+* The LiveCD operating system image, which includes the `pre-configured ``salt-minion`` service.
+For the image, see https://build.opensuse.org/package/show/systemsmanagement:yomi/openSUSE-Tumbleweed-Yomi
+
+
+To use Yomi for installing your client operating system, you need to follow this process:
+
+* Install and configure the ``salt-master``service.
+* Install the Yomi formula package.
+* Prepare the Salt pillar for the new installation.
+* Boot the new systems with the operating system image, or use PXE boot.
```

Comment:
Maybe needs to be pointed that the new machine needs to boot using the Yomi image. Once the pxe-yomi-image gets merged in Uyuni, I will add the instructions.